src/main/java/Todo.java

```diff
@@ -0,0 +1,30 @@
+public class Todo extends Task {
+
+    Todo(String taskName) {
+        super(taskName);
+    }
+
+    @Override
+    public String toString() {
+        return "[T]" + super.toString();
+    }
+
+    @Override
+    public String toFileFormat() {
+        return "T|" + (this.isDone ? "1" : "0") + "|" + this.taskName;
+    }
+
+    /**
+     * Load a todo task from the storage file format.
+     *
+     * @param fileFormatString storage file format string representation of the todo task
+     */
+    static Todo fromFileFormat(String fileFormatString) {
+        String[] tokens = fileFormatString.split("\\|");
+        Todo loaded = new Todo(tokens[2]);
```

Comment:
tokens is a good name for the this case. Not so sure about loaded though, although we know that it represents your loaded task after reading the code, but it's not intuitive at first. 🤔